script.js

```diff
@@ -77,3 +78,102 @@ function myFunction(x) {
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveVersion: 'https://samiullah997.github.io/Portfolio-Mobile/',
+    liveSource: 'https://github.com/samiullah997/Portfolio-Mobile',
+  },
+  {
+    title: 'Multi-Post Stories',
+    image: './images/img-placeholder-1.png',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveVersion: 'https://samiullah997.github.io/Portfolio-Mobile/',
+    liveSource: 'https://github.com/samiullah997/Portfolio-Mobile',
+  },
+  {
+    title: 'Multi-Post Stories',
+    image: './images/img-placeholder-1.png',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveVersion: 'https://samiullah997.github.io/Portfolio-Mobile/',
+    liveSource: 'https://github.com/samiullah997/Portfolio-Mobile',
+  },
+  {
+    title: 'Multi-Post Stories',
+    image: './images/img-placeholder-1.png',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveVersion: 'https://samiullah997.github.io/Portfolio-Mobile/',
+    liveSource: 'https://github.com/samiullah997/Portfolio-Mobile',
+  },
+  {
+    title: 'Multi-Post Stories',
+    image: './images/img-placeholder-1.png',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveVersion: 'https://samiullah997.github.io/Portfolio-Mobile/',
+    liveSource: 'https://github.com/samiullah997/Portfolio-Mobile',
+  },
+  {
+    title: 'Multi-Post Stories',
+    image: './images/img-placeholder-1.png',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveVersion: 'https://samiullah997.github.io/Portfolio-Mobile/',
+    liveSource: 'https://github.com/samiullah997/Portfolio-Mobile',
+  },
+];
+
+const setCardData = document.getElementById('recent-frame-2');
+showItemInfo.forEach((itemsData, position) => {
+  let content = `<div class="card-work media-card-${position + 1}" id="card-work">`;
+  content += '<div class="post-right-block">';
+  content += `<h1 class="card-post-title" id="post-title">${itemsData.title}</h1>`;
+  content += '<p class="supporting-text" id="post-desc">';
+  content += ` ${itemsData.desc}`;
+  content += '</p>';
+  content += '<ul class="post-tags-2">';
+  for (let i = 0; i < itemsData.technologies.length; i + 1) {
+    content += `<li id="post-tech${i}}">${itemsData.technologies[i]}</li>`;
```

Comment:
- [ ] Good job on implementing the previous reviewer's requested changes, +1:  but there are some issues you need to address :disappointed: your popup is not loading because we are getting some errors, Please fix them in order to make your project works correctly :smile: you can check the screencast below :point_down: 

[Screencast from 07-20-2022 09:47:22 PM.webm](https://user-images.githubusercontent.com/6615394/180089349-e3921ebc-4b94-4ec6-824e-9ee3c751de8e.webm)

Reply:
Hi @samiullah997, Your project wasn't approved, I made a mistake. Please work on the changes request above and submit it again :smile: